File: 5-Leccion-Arreglos-matrices/B.Ignorando-los-ultimos-elementos.cs
using System.Collections.Generic;
using System.Linq;
using System;

class Program
{
  static void Main(string[] args)
  {
    int N = int.Parse(Console.ReadLine());
    int[] arr = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();
    int K = int.Parse(Console.ReadLine());

    for (int i = 0; i < N - K; i++) {
      bool esElUltimo = i == N - K - 1;
      Console.Write(string.Format("{0}{1}", arr[i], (esElUltimo ? "\n" : " ")));
    }
  }
}

File: 5-Leccion-Arreglos-matrices/B.Ignorando-los-ultimos-elementos.cpp
#include <iostream>
using namespace std;

int main() {
  int N;
  cin >> N;
  int arr[N];

  for (int i = 0; i < N; i++)
    cin >> arr[i];

  int K;
  cin >> K;

  for (int i = 0; i < N - K; i++) {
    bool esElUltimo = i == N - K - 1;
    cout << arr[i] << (esElUltimo ? '\n' : ' ');
  }
}